README.md

```diff
@@ -18,39 +15,43 @@
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+**To get started with to project and to get a local copy up and running follow these simple example steps run :**
```

Comment:
- [ ] Please, kindly add the required rails `db:create, rails db:migrate, and rails db:seed` to your readme instructions so your users can run your app properly.